test/JDBC/expected/Test-sp_babelfish_volatility-vu-verify.out

```diff
@@ -395,9 +395,10 @@ test_sp_babelfish_volatility_schema2#!#test_sp_babelfish_volatility_f1#!#immutab
 /* test with duplicate function name */
 exec sys.sp_babelfish_volatility 'test_sp_babelfish_volatility_duplicate'
```

Comment:
this test has now two variants. One via upgrade where we will still get the error. Other is for latest which is no longer valid. We should keep the upgrade variant at-least.

Reply:
This test already has an upgrade variant.